pkg/apis/eksctl.io/v1alpha5/defaults.go

```diff
@@ -230,19 +231,21 @@ func getDefaultVolumeType(nodeGroupOnOutposts bool) string {
 	return DefaultNodeVolumeType
 }
 
-func setContainerRuntimeDefault(ng *NodeGroup, apiVersion string) {
+func setContainerRuntimeDefault(ng *NodeGroup, clusterVersion string) {
 	var runtime string
 	if ng.ContainerRuntime != nil {
 		return
 	}
 
-	if apiVersion < DockershimDeprecationVersion {
+	isDockershimDeprecated, _ := utils.IsMinVersion(DockershimDeprecationVersion, clusterVersion)
```

Comment:
We try not to ignore errors but this will require changing the signature of all callers, so I think we can leave it for now.